p2p/raintree/network.go

```diff
@@ -162,7 +162,7 @@ func (n *rainTreeNetwork) HandleNetworkData(data []byte) ([]byte, error) {
 
 	// Avoids this node from processing a messages / transactions is has already processed at the
 	// application layer. The logic above makes sure it is only propagated and returns.
-	// TODO(team): Add more tests to verify this is sufficient for deduping purposes.
+	// TODO (#278): Add more tests to verify this is sufficient for deduping purposes.
```

Comment:
Perhaps DISCUSS #278 and then TODO (issue-#) would be more appropriate

Reply:
Agreed

```suggestion
	// DISCUSS(#278): Add more tests to verify this is sufficient for deduping purposes.
```